backend/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+)
+
+type Message struct {
+	UserId   uint   `json:"userId"`
+	UserName string `json:"userName"`
+	Content  string `json:"content"`
+	Time     int64  `json:"time"`
+}
+
+func main() {
+	messageList := []Message{
+		{
+			UserId:   2,
+			UserName: "たろう",
+			Content:  "あ〜〜〜〜〜",
+			Time:     1617371835000,
+		},
+		{
+			UserId:   1,
+			UserName: "じろう",
+			Content:  "う〜〜〜",
+			Time:     1617372835000,
+		},
+		{
+			UserId:   2,
+			UserName: "たろう",
+			Content:  "画像が入る",
+			Time:     1617373835000,
+		},
+	}
+
+	http.HandleFunc("/messages", func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "GET" {
+			w.WriteHeader(405)
+			return
+		}
+		w.Header().Set("Content-Type", "application/json")
+		messageJson, _ := json.Marshal(messageList)
+		fmt.Fprintf(w, string(messageJson))
+	})
+	http.HandleFunc("/message", func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == "POST" {
+			body, _ := ioutil.ReadAll(r.Body)
+			var message Message
```

Comment:
個人的には `:=` を使うほうがいいかなと思います
```suggestion
			message := Meesage{}
```

Reply:
suggestonありがとうございます！
typoをこちらで修正しました:pray: https://github.com/gijutsu-benkyo-suru-kai/chat/pull/5/commits/e69e0546b85e79a3989b7d384908e81a3d8e4aba